### PR TITLE
Bootstrap Docker and Add README.md

### PR DESCRIPTION
**Pull request by shmashing**
This commit bootstraps the docker builds for the service and adds the README that tells how to do allllll the things

### CODE REVIEW

**Comment by shmashing** [comment]
@jaredonline ready for your 👀

File: README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Destiny Scrims Tracker #
+Version 1.0.0
```
Comment (jaredonline) [inline README.md]: No way we're at `1.0.0` yet hombre! More like `0.0.1`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Does you and your clan run a lot of private match scrimmages? Do you have a hard time trying to match up teams to be as fair as possible and still be a good fight?! Well then this is the tool for you! Destiny Scrims Tracker is a web interface to manage all of your internal clan scrimmaging and ranking! With this, all you need to do is input the results of a game and it will calculate a ELO system specific to your clan!
+
+### Models
+
+- [Guardian Object](Destiny.ScrimsTracker.Logic/Models/Guardian.cs)
+- [GuardianMatchResult Object](Destiny.ScrimsTracker.Logic/Models/GuardianMatchResult.cs)
+- [Match Object](Destiny.ScrimsTracker.Logic/Models/Match.cs)
```
Comment (jaredonline) [inline README.md:10]: These don't actually exist yet, right?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+#### Quickstart Examples
+
+The following snippets are provided for illustrative purposes only and the tests should be referred to as the source of "implementation truth".
+
+##### Starting the Service Locally
+
+To build the service locally, from the root of the repository, run `dotnet build`.
+
+To run the service locally, from the root of the repository, run 'dotnet run --project Destiny.ScrimsTracker.Api/Destiny.ScrimsTracker.Api.csproj'
```
Comment (jaredonline) [inline README.md]:
```suggestion
To run the service locally, from the root of the repository, run `dotnet run --project Destiny.ScrimsTracker.Api/Destiny.ScrimsTracker.Api.cspro`'
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+To run the tests locally, from the root of the repository, run `dotnet test`.
+
+## Database Migration
+
+This repository **will** include a DbUp console app for executing database migrations. Simply add a script to `Destiny.ScrimsTracker.Logic/Migrations/` and set the file as an embedded resource via the .csproj file.
+
+To run the migration, execute the console app with the following command:
+`dotnet run -s <server endpoint> -d <database name> -p <port> -u <user id> -w <password>`
+
```
Comment (jaredonline) [inline README.md:49]: Can me Makify these commands as:

- `make build`
- `make build_ds`
- `make run`
- `make run_ds`
- `make test`
- `make run_migrations` - this one might be hard with the need for the password and ish

It's fine if this happens in a later PR

**Review by jaredonline — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (shmashing) [reply README.md]: waaat

File: README.md (hunk 2 above)
Comment (shmashing) [reply README.md:10]: They do!

File: README.md (hunk 4 above)
Comment (shmashing) [reply README.md:49]: yea! I'm unfamiliar with all that tho

File: README.md (hunk 1 above)
Comment (jaredonline) [reply README.md]: 1.0.0 implies a solid interface and functionality. 0.0.1 implies things are changing rapidly, don't rely on anything.

File: README.md (hunk 2 above)
Comment (jaredonline) [reply README.md:10]: Ah, just found them. Can they get deleted and re-added via a PR. I have thoughts.

File: README.md (hunk 4 above)
Comment (jaredonline) [reply README.md:49]: I can do it once this is merged

File: README.md (hunk 1 above)
Comment (shmashing) [reply README.md]: 😂 na i know just forgot to change it

File: README.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+#### Quickstart Examples
+
+The following snippets are provided for illustrative purposes only and the tests should be referred to as the source of "implementation truth".
+
+##### Starting the Service Locally
+
+To build the service locally, from the root of the repository, run `dotnet build`.
+
+To run the service locally, from the root of the repository, run `dotnet run --project Destiny.ScrimsTracker.Api/Destiny.ScrimsTracker.Api.cspro`'
```
Comment (jaredonline) [inline README.md]:
```suggestion
To run the service locally, from the root of the repository, run `dotnet run --project Destiny.ScrimsTracker.Api/Destiny.ScrimsTracker.Api.cspro`
```

**Review by jaredonline — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (shmashing) [reply README.md:10]: Mostly they just there for placeholders. Or did you have thoughts on where they should be?

Comment (jaredonline) [reply README.md:10]: No, on their contents and which models we should have. I have no idea where things should be in a .NET project

Comment (shmashing) [reply README.md:10]: Done

**Review by jaredonline — APPROVED** [review]

**Pull request merged**